src/server/middleware/sendHtml.js

```diff
@@ -69,30 +81,36 @@ export function renderStaticErrorPage(res) {
     message = 'Sorry, we are unable to load this page at this time.';
   }
 
-  // TODO: allow root module to provide custom error message and override default html
-  safeSend(res,
-    `<!DOCTYPE html>
-        <html>
-          <head>
-            <title>One App</title>
-            <meta http-equiv="X-UA-Compatible" content="IE=edge">
-            <meta charset="utf-8">
-            <meta name="viewport" content="width=device-width, initial-scale=1">
-            <meta name="application-name" content="one-app">
-          </head>
-          <body style="background-color: #F0F0F0">
-            <div id="root">
-              <div>
-                <div style="width: 70%; background-color: white; margin: 4% auto;">
-                  <h2 style="display: flex; justify-content: center; padding: 40px 15px 0px;">Loading Error</h2>
-                  <p style="display: flex; justify-content: center; padding: 10px 15px 40px;">
-                    ${message}
-                  </p>
-                </div>
+  const fallbackUrl = process.env.ONE_FALLBACK_URL;
+
+  const errorResponse = `<!DOCTYPE html>
+      <html>
+        <head>
+          <title>One App</title>
+          <meta http-equiv="X-UA-Compatible" content="IE=edge">
+          <meta charset="utf-8">
+          <meta name="viewport" content="width=device-width, initial-scale=1">
+          <meta name="application-name" content="one-app">
+        </head>
+        <body style="background-color: #F0F0F0">
+          <div id="root">
+            <div>
+              <div style="width: 70%; background-color: white; margin: 4% auto;">
+                <h2 style="display: flex; justify-content: center; padding: 40px 15px 0px;">Loading Error</h2>
+                <p style="display: flex; justify-content: center; padding: 10px 15px 40px;">
+                  ${message}
+                </p>
               </div>
             </div>
-          </body>
-        </html>`);
+          </div>
+        </body>
+      </html>`;
+
+  if (fallbackUrl) {
+    await fetchCustomErrorPage(fallbackUrl, res);
+  } else {
+    safeSend(res, errorResponse);
```

Comment:
can you you fetchCustomErrorPage to get the response and pass into safeSend here

Reply:
Any particular reason to not do it `fetchCustomErrorPage`?